config/sync/user.role.site_editor.yml

```diff
@@ -23,6 +23,7 @@ permissions:
   - 'administer nodes'
   - 'administer stanford-event-types menu items'
   - 'administer stanford-person-type menu items'
+  - 'create embeddable media'
```

Comment:
Shouldn't the contributor role also be able to create embeddable media?

Reply:
when we had the conversation about permissions, I recall we decided that was not going to be the case, but if that's different at this point, it's no big deal to change it.

Reply:
That is fine with me. Keep it this way.